qa/L0_backend_python/test.sh

```diff
@@ -132,296 +133,314 @@ mkdir -p models/dlpack_identity/1/
 cp ../python_models/dlpack_identity/model.py ./models/dlpack_identity/1/
 cp ../python_models/dlpack_identity/config.pbtxt ./models/dlpack_identity
 
-# Skip torch install on Jetson since it is already installed.
+# Env test should be run first so we can get the stubs for each of the
+# environments
+# Disable env test for Jetson since cloud storage repos are not supported
 if [ "$TEST_JETSON" == "0" ]; then
-  pip3 install torch==1.13.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
-else
-  # GPU tensor tests are disabled on jetson
-  EXPECTED_NUM_TESTS=9
+    # In 'env' test we use miniconda for dependency management. No need to run
+    # the test in a virtual environment.
+    (cd env && bash -ex test.sh)
+    if [ $? -ne 0 ]; then
+        echo "Subtest env FAILED"
+        RET=1
+    fi
 fi
 
-prev_num_pages=`get_shm_pages`
-run_server
-if [ "$SERVER_PID" == "0" ]; then
-    cat $SERVER_LOG
-    echo -e "\n***\n*** Failed to start $SERVER\n***"
-    exit 1
-fi
+for PYTHON_ENV in ${PYTHON_ENV_VERSION}; do
```

Comment:
I was wondering why do we need to iterate on all the Python versions? I think we want separate jobs that create a stub and setup the environment based on the PYTHON_ENV_VERSION listed. Then, we only need to change the environment version for each job to use a different Python version.

We would have 3 different variants of L0_backend_python for testing different Python versions (`L0_backend_python--py3.x`) + the default L0_backend python. Each job should only test a single Python version with the exception of the `env` subset.

Reply:
we are not. We put them in one at a time on the CI.